lldb/source/Core/Mangled.cpp

```diff
@@ -148,6 +148,41 @@ void Mangled::SetValue(ConstString name) {
   }
 }
 
+// BEGIN SWIFT
+#ifdef LLDB_ENABLE_SWIFT
+static ConstString GetSwiftDemangledStr(ConstString m_mangled,
+                                        const SymbolContext *sc,
+                                        ConstString &m_demangled) {
+  const char *mangled_name = m_mangled.GetCString();
```

Comment:
Not sure what the default behavior is, but I think we need something like this

```suggestion
  const char *mangled_name = m_mangled.GetCString("");
```

Reply:
Probably `m_mangled.AsCString("")`